docs/src/examples.rst

```diff
@@ -14,205 +14,175 @@
+                super(parent, name, props);
 
-   // Production stacks
-   const prod = [
-     new MyStack(app, {
-       name: 'NAEast',
-       env: { region: 'us-east-1' }
-   }),
 
-   new MyStack(app, {
-     name: 'NAWest',
-     env: { region: 'us-west-2' }
-   }),
+              }
+            }
 
-   new MyStack(app, {
-     name: 'EU',
-     env: { region: 'eu-west-1' },
-       encryptedStorage: true
-     })
-   ]
+            const app = new cdk.App();
 
-   // CI/CD pipeline stack
-   new DeploymentPipeline(app, {
-     env: { region: 'us-east-1' },
-      strategy: DeploymentStrategy.Waved,
-      preProdStages: [ preProd ],
-      prodStages: prod
-   });
+            new MyEcsConstructStack(app, 'MyEcsConstructStack');
 
-   app.run();
+            app.run();
 
-.. _dynamodb_example:
+        Save it and make sure it builds and creates an empty stack.
 
-Creating a |DDB| Table
-======================
+        .. code-block:: sh
 
-The following example creates a
-|DDB| table with the partition key **Alias**
-and sort key **Timestamp**.
+            npm run build
+            cdk synth
 
-.. code-block:: js
+        You should see a stack like the following,
+        where CDK-VERSION is the version of the CDK.
 
-   import dynamodb = require('@aws-cdk/aws-dynamodb');
-   import cdk = require('@aws-cdk/cdk');
+        .. code-block:: sh
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+            Resources:
+              CDKMetadata:
+                Type: 'AWS::CDK::Metadata'
+                Properties:
+                  Modules: @aws-cdk/cdk=CDK-VERSION,@aws-cdk/cx-api=CDK-VERSION,my_ecs_construct=0.1.0
 
-       const table = new dynamodb.Table(this, 'Table', {
-         tableName: 'MyAppTable',
-         readCapacity: 5,
-         writeCapacity: 5
-       });
+.. _creating_ecs_l2_example_2:
 
-       table.addPartitionKey({ name: 'Alias', type: dynamodb.AttributeType.String });
-       table.addSortKey({ name: 'Timestamp', type: dynamodb.AttributeType.String });
-     }
-   }
+Step 2: Add the |EC2| and |ECS| Packages
+----------------------------------------
 
-   const app = new cdk.App();
+Install support for |EC2| and |ECS|.
 
-   new MyStack(app, 'MyStack');
+.. tabs::
 
-    app.run();
+    .. group-tab:: TypeScript
 
-.. _creating_rds_example:
+        .. code-block:: sh
 
-Creating an |RDS| Database
-==========================
+            npm install @aws-cdk/aws-ec2 @aws-cdk/aws-ecs
 
-The following example creates the Aurora database **MyAuroraDatabase**.
+.. _creating_ecs_l2_example_3:
 
-.. code-block:: js
+Step 3: Create a Fargate Service
+--------------------------------
 
-   import ec2 = require('@aws-cdk/aws-ec2');
-   import rds = require('@aws-cdk/aws-rds');
-   import cdk = require('@aws-cdk/cdk');
+There are two different ways of creating a serverless infrastructure with |ECS|:
```

Comment:
I would write "two different ways of running your container tasks"

Reply:
ok